gulpfile.mjs

```diff
@@ -2099,6 +2099,35 @@ gulp.task("lint", function (done) {
   });
 });
 
+gulp.task(
+  "lint-mozcentral",
```

Comment:
This doesn't seem to be run as part of our CI at the moment. Is that intentional, and and will it be triggered somewhere in a mozcentral pipeline, or should it be added here too?